### ISSUE TITLE
Replace react-autocomplete with react-autosuggest

### ISSUE THREAD

**Issue by julianguyen** [ifmeorg/ifme#1545]
# Description 

[react-autocomplete](https://github.com/reactjs/react-autocomplete) is no longer being maintained. 

After updating our client to the latest version of React, we receive these warnings:
```
      Warning: componentWillMount has been renamed, and is not recommended for use. See https://fb.me/react-async-component-lifecycle-hooks for details.

      * Move code with side effects to componentDidMount, and set initial state in the constructor.
      * Rename componentWillMount to UNSAFE_componentWillMount to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

      Please update the following components: Autocomplete
    console.warn node_modules/react-dom/cjs/react-dom.development.js:11494
      Warning: componentWillReceiveProps has been renamed, and is not recommended for use. See https://fb.me/react-async-component-lifecycle-hooks for details.

      * Move data fetching code or side effects to componentDidUpdate.
      * If you're updating state whenever props change, refactor your code to use memoization techniques or move it to static getDerivedStateFromProps. Learn more at: https://fb.me/react-derived-state
      * Rename componentWillReceiveProps to UNSAFE_componentWillReceiveProps to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

      Please update the following components: Autocomplete
```

We could hypothetically fork `react-autocomplete` and update React ourselves and use it in our client, but that creates an maintenance overhead for us. The best course of action is to use another framework. 

Let's use [react-autosuggest](https://github.com/moroshko/react-autosuggest) instead!

You'll want to update `client/app/components/Input/InputTag.jsx`.

---

Please assign yourself (via the Assignees dropdown), if you do want to work on this issue. Can't find yourself? You need to [join our organization](https://github.com/ifmeorg/ifme/wiki/Join-Our-Slack).

Check out our [Picking Up Issues](https://github.com/ifmeorg/ifme/wiki/Picking-Up-Issues) guide if you haven't already!

**Comment by CSumm** [comment]
Is anyone assigned to this issue? I'd like to take a shot at it if it's still available

**Comment by neon98** [comment]
hey @CSumm are you working on this issue? I'd love to help!

**Comment by CSumm** [comment]
I've written some code for it, but I'm hitting a roadblock with it. I can send you what I've done so far and maybe you can take a look when you have the chance.

**Comment by neon98** [comment]
Sure. I'll look into it :D

**Comment by CSumm** [comment]
@neon98 take a look at the pull request I just did which is the work in progress. It has the code I'm trying to use and you can copy it into your local repo.

**Comment by CSumm** [comment]
I have to jump ship on this issue as life came up. Neon98 will take over from here

**Comment by Rajani169** [comment]
Hey!! Can you please assign this issue to me?

**Comment by julianjohannesen** [comment]
Quick question. When setting up the environment, is it okay to use the most recent version of PostgreSQL, or should we stick to a particular version?

**Comment by julianguyen** [comment]
@Rajani169, you'll need to [join](https://github.com/ifmeorg/ifme/wiki/Join-Our-Slack) our contributor community in order to be able to assign issues to yourself. 

@julianjohannesen The most recent version should work, if not let us know!

**Comment by deschantkn** [comment]
I would love to take on this task if possible!

**Comment by SonuToor** [comment]
> Quick question. When setting up the environment, is it okay to use the most recent version of PostgreSQL, or should we stick to a particular version?

Hey @julianjohannesen are you still working on this issue?

**Comment by SonuToor** [comment]
Hey guys, I'm going to have a crack at this. I'll update here or on the slack channel if I run into any issues!
